### DISCUSSION TITLE
Spin Multiplicity or Number of Unpaired  Electrons Option?

### DISCUSSION THREAD

**Discussion by wuwenkun** [dftd4/dftd4#270] [Q&A] [answered]
Hello,

I have installed DFTD4 and find the options include: 

-c,--charge <real>       Set charge to molecule, overwrites .CHRG file
-i,--input <format>      Hint for the format of the input file
-f,--func <method>       Use damping parameters for given functional
   --param <list>        Specify parameters for rational damping,
                         expected order is s6, s8, a1, a2 (requires four arguments)
   --mbdscale <s9>       Use scaled ATM three-body dispersion
   --zeta <list>         Adjust charge scaling parameters, takes two reals,
                         expected order is ga, gc (default: 3.0, 2.0)
   --wfactor <real>      Adjust weighting factor for interpolation
                         (default: 6.0)
-g,--grad [file]         Evaluate molecular gradient and virial,
                         write results to file (default: dftd4.txt),
                         attempts to add to Turbomole gradient and gradlatt files
   --hessian             Evaluate molecular hessian
   --property            Show dispersion related atomic and system properties
   --pair-resolved       Calculate pairwise representation of dispersion energy
   --noedisp             Disable writing of dispersion energy to .EDISP file
   --json [file]         Dump results to JSON output (default: dftd4.json)
-v,--verbose             Show more, can be used multiple times
-s,--silent              Show less, use twice to supress all output
   --version             Print program version and exit
   --citation            Print citation information and exit
   --license             Print license header and exit
-h,--help                Show this help message

Here I only see the charge option. I assume that we expect to see a difference with varying number of unpaired electrons in the system for dftd4 calculations. Is there a spin multiplicity option since they can not be included in the .xyz file?

Regards,
Wenkun

**Comment by awvwgk** [comment, marked as answer]
In D4 the spin multiplicity is not accounted for only the total charge.

**Reply by wuwenkun** [reply]
Thank you for your reply. I did a bit more search myself and found "The third integer contains the total charge and the fourth integer the spin as number of unpaired electrons. The total charge and the systems spin are stored in the [structure_type]" for the Gaussian format input (source: https://grimme-lab.github.io/mctc-lib/page/format-ein.html). What is the spin information included here for?

**Reply by awvwgk** [reply]
Some geometry format supported by D4 can read either unpaired electrons or multiplicity. Even if provided, this information will not be used for computing the D4 energy.

**Reply by wuwenkun** [reply]
Thank you for your clarification!
